src/Text.Analyzers/Core/IdentifiersShouldBeSpelledCorrectly.cs

```diff
@@ -210,6 +211,12 @@ public sealed class IdentifiersShouldBeSpelledCorrectlyAnalyzer : DiagnosticAnal
             isPortedFxCopRule: true,
             isDataflowRule: false);
 
+        /// <summary>
+        /// Todo, use an actual AdditionalTextValueProvider once it is available:
+        /// https://github.com/dotnet/roslyn/issues/67611
+        /// </summary>
+        private readonly ConditionalWeakTable<AdditionalText, CodeAnalysisDictionary> _textToDictionary = new();
```

Comment:
>private readonly ConditionalWeakTable<AdditionalText, CodeAnalysisDictionary> _textToDictionary = new(); [](http://example.com/codeflow?start=8&length=104)

Not sure what the lifetime of this object is, but I assume this member is intentionally not static. If so, might be nice to doc that too.

Reply:
Correct.  Manish preferred non-static CWTs for additional-text mappings.